build.rs

```diff
@@ -0,0 +1,39 @@
+use std::process::Command;
+
+fn main() {
+    // We delete the old build/ directory in order to avoid mixing files up.
+    if std::path::PathBuf::from("build").exists() {
```

Comment:
We should put these files in `$OUT_DIR`, a directory created by Cargo when running build scripts - it can be obtained using `env!("OUT_DIR")`.

Then, from within source files, you can do things like `include_str!(concat!(env!("OUT_DIR"), "git_version.in"))` or whatever to read the files you placed there.